### DISCUSSION TITLE
Time series panel add possibility to select for which graph a thresholds should be shown

### DISCUSSION THREAD

**Discussion by tig987** [grafana/grafana#37339] [Dashboards, Panels & Visualizations]
In the new Time series panel thresholds are applied generally to all graphs.
This works fine when only one y-scale is used.
But when for example different y-scales (left/right) for two graphs are used, each threshold is shown twice.
Since the threshold usually only makes sense for one of the graphs, it should be selectable to which scale the threshold should be applied to.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
